packages/ui/src/routes/bridge/BridgeConfirmPage.tsx

```diff
@@ -232,24 +240,44 @@ const BridgeConfirmPage: FunctionComponent<{}> = () => {
         ? hasNativeAssetBalance
         : hasNativeAssetBalance && hasFromTokenBalance
 
-    const nativeTokensInDestinationNetworkStatus =
-        useAddressHasEnoughNativeTokensToSend(toChainId, toToken.address)
+    const { hideBridgeInsufficientNativeTokenWarning } = useUserSettings()
+    const { isLoading, result, check } =
```

Comment:
Can you rename the check function to be explicit about what it does? 

`{  isLoading: isLoadingUserHasEnough..., result: userBalanceToCoverFeesInTarget..., check: checkUser... }`